### ISSUE TITLE
Highlight only selected text

### ISSUE THREAD

**Issue by cibic89** [jcb91/jupyter_highlight_selected_word#25]
Hi, is there a way to highlight only selected text? If I select a block of code it highlights words within. Have both "...match whole words" and "highlight matches if surrounded by non-word characters..." off although the latter is off.

![image](https://user-images.githubusercontent.com/22660599/34576728-ce32ebe4-f176-11e7-8654-4c3096c21e3a.png)

**Comment by jcb91** [comment]
Do you mean a way to not have the little lines between each word? Unfortunately I haven't been able to think of a way to do this yet 😞
The problem is that the highlighting is accomplished by CodeMirror applying CSS classes to existing elements, which are all separate. As a result, a background-colour highlight works fine, but a border ends up around each individual word/element...

**Comment by cibic89** [comment]
Yes, when I switch to that the problem is gone. Works fine for me!

Sent from BlueMail<http://www.bluemail.me/r?b=11687>
On 5 Jan 2018, at 11:51, Josh Barnes <notifications@github.com<mailto:notifications@github.com>> wrote:

Do you mean a way to not have the little lines between each word? Unfortunately I haven't been able to think of a way to do this yet 😞
The problem is that the highlighting is accomplished by CodeMirror applying CSS classes to existing elements, which are all separate. As a result, a background-colour highlight works fine, but a border ends up around each individual word/element...
